src/components/common-bird/CommonBird.tsx

```diff
@@ -0,0 +1,302 @@
+import React, { FC, useState, useCallback, Fragment } from "react";
+import {
+  Button,
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  Input,
+  Label
+} from "reactstrap";
+import {
+  Autosuggest,
+  IAutosuggest,
+  IChangeValue
+} from "../autosuggest/Autosuggest";
+
+import { Map } from "../map/Map";
+
+import { labels } from "../../config/i18n/labels";
+
+import { Bird, IBird } from "../bird/Bird";
+import { InfoContainer } from "../info-container/InfoContainer";
+import { DropZone } from "../drag-n-drop/DragAndDrop";
+
+import "./CommonBird.scss";
+
+const blockName = "common-bird";
+
+interface CollectionItem {
```

Comment:
I guess it's better to move interfaces to models.ts file in order to diminish size of this CommonBird.tsx file

Reply:
fixed